pkg/controller/servicemesh/controlplane/reconciler.go

```diff
@@ -49,10 +52,30 @@ var orderedCharts = []string{
 	"istio/charts/kiali",
 }
 
+const (
+	smcpTemplatePath        = "/etc/istio-operator/templates/"
+	smcpDefaultTemplatePath = "/etc/istio-operator/default-templates/"
+	smcpDefaultTemplate     = "default"
```

Comment:
I was wondering if we could use this `default` template to replace our default-overrides that we currently do via `sed`? We'd have to make sure all those defaults end up in the other templates as well, of course.

Reply:
That's my hope here. Everything except for a couple of deletes is between base, Maistra, and product. Maistra specific stuff is what was community before. Product stuff is what was product before. Base is everything else. Product and Maistra use base as their template.

Reply:
The build process will add either Maistra or product as default.

Reply:
I like this idea.  Have you tried using `null` for the values being deleted?

Reply:
awesome!